templates/form_theme_admin.html.twig

```diff
@@ -1,5 +1,16 @@
 {% extends '@SonataDoctrineORMAdmin/Form/form_admin_fields.html.twig' %}
 
+{% block measure_entry_row %}
+    <div class="row">
+        <div class="col-xs-4">
+            {{ form_row(form.key, { required: true }) }}
+        </div>
+        <div class="col-xs-4">
+            {{ form_row(form.value, { required: true }) }}
```

Comment:
it's already required by FormType, you could remove it?

Reply:
x-files number 2!